packages/blade/src/components/Alert/Alert.stories.tsx

```diff
@@ -1,41 +1,26 @@
 import type { ComponentStory, Meta } from '@storybook/react';
-import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import { Title } from '@storybook/addon-docs';
 import type { ReactElement } from 'react';
 
 import type { AlertProps } from './Alert';
 import { Alert as AlertComponent } from './Alert';
-import useMakeFigmaURL from '~src/_helpers/storybook/useMakeFigmaURL';
 import { colors } from '~tokens/global';
 import Box from '~components/Box';
-import FigmaEmbed from '~src/_helpers/storybook/FigmaEmbed';
 import Sandbox from '~src/_helpers/storybook/Sandbox';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
 
 const Page = (): ReactElement => {
-  const figmaURL = useMakeFigmaURL([
-    {
-      themeTokenName: 'paymentTheme',
-      lightModeURL:
-        'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6922%3A17789',
-      darkModeURL:
-        'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6922%3A17789',
-    },
-    {
-      themeTokenName: 'bankingTheme',
-      lightModeURL:
-        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=11098%3A286031',
-      darkModeURL:
-        'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=11098%3A286031',
-    },
-  ]);
-
   return (
-    <>
-      <Title />
-      <Subtitle>
-        Alerts are messages that communicate information to users about any significant changes or
-        explanations inside the system in a prominent way.
-      </Subtitle>
-      <FigmaEmbed title="Alert Figma Designs" src={figmaURL} />
+    <StoryPageWrapper
+      componentName="Alert"
+      componentDescription="Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a prominent way."
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6922%3A17789',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=11098%3A286031',
+      }}
+    >
       <Title>Usage</Title>
```

Comment:
do you think we can move this to wrapper as well? since this is a static title across all the stories

Reply:
Leaving it outside wrapper leaves some space to add any other custom text and headlines inside. 

E.g. in visually hidden stories, we're adding the link to RFC before the usage title
https://github.com/razorpay/blade/blob/eab3b33ad44946909b7301a7b1d6861972e5fcca/packages/blade/src/components/VisuallyHidden/VisuallyHidden.stories.tsx#L19-L32

So I think we can leave it outside. It also looks better semantically where we say "Usage" heading first and then add Sandbox which is the actual usage.